lib/unlock-helpers.js

```diff
@@ -9,6 +9,8 @@ const PATTERN_UNLOCK = 'pattern';
 const FINGERPRINT_UNLOCK = 'fingerprint';
 const UNLOCK_TYPES = [PIN_UNLOCK, PASSWORD_UNLOCK, PATTERN_UNLOCK, FINGERPRINT_UNLOCK];
 const KEYCODE_NUMPAD_ENTER = '66';
+const KEYCODE_POWER = '26';
+const KEYCODE_WAKEUP = '224'; // Can work over API Level 20
```

Comment:
key codes may be integers

Reply:
Either worked fine. I defined as string because of the previous implementation.
I ensured this worked, including `KEYCODE_NUMPAD_ENTER`, worked as _int_ as well.